tla/consistency/ExternalHistoryInvars.tla

```diff
@@ -88,6 +88,14 @@ OnceCommittedPrevCommittedInv ==
         /\ history[j].tx_id[2] <= history[i].tx_id[2]
         => history[j].status = CommittedStatus
 
+\* If a transaction is committed then all others from greater seqnums but smaller terms are invalid
+OnceCommittedNextInvalidInv ==
+    \A i, j \in TxStatusReceivedEventIndexes:
+        /\ history[i].status = CommittedStatus
+        /\ history[i].tx_id[2] <= history[j].tx_id[2]
```

Comment:
Not in the scope of this PR, but consider using a record instead of a tuple to model `tx_id` as part of consolidating/refactoring the specs; `tx_id.term` and `tx_id.seqno` read much nicer.

Reply:
I actually did this for the tla+ snippets (@achamayou suggestion I believe) and it helped alot with readability